### ISSUE TITLE
Cannot find the ESP partition mount point.

### ISSUE THREAD

**Issue by mabod** [systemd/systemd#11950]
**systemd version the issue has been seen with**
systemd 241 (241.588-1-manjaro)

**Used distribution**
Manjaro testing

**Expected behaviour you didn't see**
I didnt see a clean journal. 

**Unexpected behaviour you saw**
I am seeing a new error message in the journal:

`Mär 10 10:18:27 rakete systemd-logind[1736]: Cannot find the ESP partition mount point.`

I see this several times during boot and when I log in. This is new. I believe it was introduced with the updates I did yesterday:
```
[2019-03-02 13:12] [ALPM] upgraded systemd-libs (241.258-1 -> 241.455-1)
[2019-03-02 13:12] [ALPM] upgraded systemd (241.258-1 -> 241.455-1)
[2019-03-02 13:13] [ALPM] upgraded lib32-systemd (241.258-1 -> 241.455-1)
[2019-03-02 13:14] [ALPM] upgraded systemd-resolvconf (241.258-1 -> 241.455-1)
[2019-03-02 13:14] [ALPM] upgraded systemd-sysvcompat (241.258-1 -> 241.455-1)
```
Other than this error message I do not see bad behavior. The PC is booting fine. 

This is a legacy boot system with grub 2.03.4. No UEFI.

**Comment by poettering** [comment]
We probably should downgrade this warning to debug.

**Comment by poettering** [comment]
Fix waiting in #11979
